### ISSUE TITLE
Can't start Messer: ": No such file or directory"

### ISSUE THREAD

**Issue by vincelwt** [mjkaufer/Messer#12]
Hi!
When I try to run messer, I have the following error.
I installed it via sudo npm install -g messer

```
vince@elementary-laptop:~/Downloads/messer$ messer
: No such file or directory
vince@elementary-laptop:~/Downloads/messer$ node -v
v6.2.0
vince@elementary-laptop:~/Downloads/messer$ npm -v
3.8.9
vince@elementary-laptop:~/Downloads/messer$ lsb_release -a
No LSB modules are available.
Distributor ID: elementary OS
Description:    elementary OS Freya
Release:    0.3.2
Codename:   freya

```

**Comment by tomquirk** [comment]
@vincelwt should be fixed as of `bcdbf40` 💃

**Comment by vincelwt** [comment]
Thanks!
